### ISSUE TITLE
[Adaptive Card > OrderDelivery.Json]: Voice over does convey the incorrect information as 'Dimmed' for the 'Delivery' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#50]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "OrderDelivery.json" control and double tap to activate it.
7. Navigate to the 'Delivery' button and observe the issue. 

**Actual Result:**
Voice over does convey the incorrect information as 'Dimmed' for the Delivery button and on double tapping on it no action is getting performed.
Also, Delivery button does not looks like a disabled control.

**Expected Result:**
Voice over should convey the correct information for the 'Delivery' button and when focus comes on it then it should announce as 'Delivery button'.

**User Experience:**
It will impact the screen reader user if the voice over does convey the incorrect information for the non-disabled control.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/b3f1d276-f829-4fcb-a5d5-c67f805e56e1

**Comment by GabrielMedAlv** [comment]
Currently we don't have any specific appearance when the button is not enabled (like the grey color for "destructive" sentiment components). By default, disabled system buttons doesn't change at all. Any change is easy to add but I have doubt of what is expected in this case (As an example I changed the alpha value when is not enabled in the screenshot). Note, this could change diff tests. cc @VikrantSingh01    

![IMG_0193](https://github.com/microsoft/AdaptiveCards-Mobile/assets/130393016/9d9f6ad3-40ad-432e-9f8e-887baf50f946)

**Comment by ardlank** [comment]
![Screenshot_20240125_135216](https://github.com/microsoft/AdaptiveCards-Mobile/assets/23054357/7a40d8cf-ba0c-4e96-8022-6e850fe66e84)
@GabrielMedAlv see this image for Android
